examples/cms/src/components/Spinner.tsx

```diff
@@ -0,0 +1,40 @@
+import cx from "classnames"
+import styles from "./spinner.module.css"
+
+export interface SpinnerProps {
+    /** Size of the spinner */
+    size?: "normal" | "medium" | "large"
+    /** Set the spinner to have a static position inline with other content */
+    inline?: boolean
+    className?: string
+    inheritColor?: boolean
+}
+
+function styleForSize(size: SpinnerProps["size"]) {
+    switch (size) {
+        case "normal":
+            return styles.normalStyle
+        case "medium":
+            return styles.mediumStyle
+        case "large":
+            return styles.largeStyle
+    }
```

Comment:
Nit. I would make the return type explicit here, that way you would get a type error if the size type is extended

Reply:
I'm not sure on how to do that as type of `styles.normalStyle` would be `string`

Reply:
```ts
function styleForSize(size: SpinnerProps["size"]): string {
```

Would this work?

Reply:
Oh, I thought we wanted something else than string

Reply:
Oh now, I just want to make sure you get a type error when adding a new size that isn't handled yet